compiler-rt/lib/sanitizer_common/sanitizer_symbolizer_markup_constants.h

```diff
@@ -26,20 +26,23 @@ constexpr const char *kFormatDemangle = "{{{symbol:%s}}}";
 constexpr uptr kFormatDemangleMax = 1024;  // Arbitrary.
 
 // Function name or equivalent from PC location.
-constexpr const char *kFormatFunction = "{{{pc:%p}}}";
```

Comment:
if uptr used for pointer, it should be `%p` but printf arg should be caster to (void*)
if it 's uptr for size, then `%zx` (for hex), or `%zu` should be used